docs/job-scheduling.md

```diff
@@ -101,6 +101,11 @@ then set `yarn.nodemanager.aux-services.spark_shuffle.class` to
 `spark.shuffle.service.*` [configurations](configuration.html).
 4. Restart all `NodeManager`s in your cluster.
 
+If you wish to use the external shuffle service in a Mesos deployment you need to launch the
+service on each Mesos slave that might run Spark jobs. A launcher script is provided in the `sbin/`
+directory: `bin/start-shuffle-service.sh`. You can later stop the service by running
+`sbin/stop-shuffle-service.sh`. Additional configuration options can be passed in `SPARK_SHUFFLE_OPTS`.
+
```

Comment:
Wait, actually my comment wasn't referring specifically to `SPARK_SHUFFLE_OPTS`. I was referring to the whole paragraph, which is currently under the Dynamic Allocation section even though DA for Mesos isn't supported yet.